### ISSUE TITLE
Typescript definition - Wrong hash() return type

### ISSUE THREAD

**Issue by soimy** [oliver-moran/jimp#745]
# Expected Behavior

According to the API document, we shall just use
``` typescript
let hashString: string = image.hash();
```
Or according to the `jimp.d.ts`
``` typescript
image.hash(hash => {
    hashString = hash;
};
```

# Current Behavior

`image.hash()` return `Jimp` object. And the callback function return a type `null | Error`

- Jimp Version: 0.6.4
- Operating System: macOS 10.14.5
- Node version: node v11.3.0

**Comment by soimy** [comment]
Seems that it's a bug in `jimp.d.ts` where `hash()` should return `string` not `this`.
Manually change the file solves the problem.

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in v0.6.5 :rocket:
